src/models/db/transfers.js

```diff
@@ -90,6 +90,20 @@ function * getTransfer (uuid, options) {
     })
 }
 
+function * getTransferByExecutionCondition (executionCondition, options) {
+  return db.selectOne({EXECUTION_CONDITION: executionCondition}, options && options.transaction)
```

Comment:
If you're going to query the db by execution condition, please add an index to the `EXECUTION_CONDITION` column in `src/sql/pg/create.sql` and `src/sql/strong-oracle/create.sql` (we don't really care about performance on sqlite), as done for `TRANSFER_UUID`.